src/pages/docs/liveobjects/counter.mdx

```diff
@@ -87,21 +100,23 @@ try counter.subscribe { update, _ in
     print("Update details: \(update)")
 }
 ```
+
+```java
+counter.subscribe((counterUpdate) -> {
+    System.out.println("Counter updated: " + counter.value());
+    System.out.println("Update details: " + counterUpdate.getUpdate());
+});
+```
 </Code>
 
 The update object provides details about the change, such as the amount by which the counter value was changed.
 
-It may also include the client ID of the client that made the change, if the change can be attributed to a specific client. For example, the client ID may be missing if the update was triggered by data resynchronization after a disconnection and the change occurred while the client was offline.
-
-Example structure of an update object when the counter was incremented by 5 by a client with the ID `my-client`:
+Example structure of an update object when the counter was incremented by 5:
 
 <Code>
 ```json
 {
-  "update": {
-    "amount": 5
-  },
-  "clientId": "my-client"
```

Comment:
This was recently added in https://github.com/ably/docs/pull/2789. Did you mean to delete it? I wouldn't have thought the structure would differ.

Reply:
I didn't mean to delete it, I though we deleted it on `main` branch so `swift` branch was showing some diff

Reply:
Will revert this change as per `main`

Reply:
updated